.github/CODEOWNERS

```diff
@@ -32,6 +32,7 @@
 # TODO: /README.md
 
 # Runtimes
+/Runtimes/                                 @etcwilde
```

Comment:
@etcwilde Are you OK with this?

Reply:
I don't think that this is right. This line should remain as is. If you want to add coverage for the files which should never be checked in, @swiftlang/standard-librarians would be the right owner.

Reply:
For now, sure. Once the stdlib has moved over entirely, that should likely end up being the stdlib librarians. I can put up a PR once that move happens.

Reply:
Sounds good, thanks.